portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/contentsecuritypolicy/ContentSecurityPolicy.testcase

```diff
@@ -0,0 +1,80 @@
+@component-name = "portal-security"
+definition {
+
+	property ci.retries.disabled = "true";
+	property portal.release = "true";
```

Comment:
Change to property portal.release = "false"; because the new feature hasn't been released.
And need to enable the feature flag: feature.flag.LPS-134060=true.